src/Files.App/Files.App.csproj

```diff
@@ -79,6 +79,7 @@
         <PackageReference Include="Microsoft.Extensions.Hosting" Version="8.0.0" />
         <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0" />
         <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.3233" />
+        <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
But why 😂

Reply:
Can't get WebViewExtsnsions to work with System.Text :(

Reply:
I tried converting the code to `System.Text.Json` but ran into an issue where the terminal text wouldn't display.

Reply:
Done!